code/game/objects/items/binoculars.dm

```diff
@@ -122,9 +125,23 @@
 	else
 		. += "binoculars_laser"
 
+/obj/item/binoculars/tactical/proc/check_mortar_index()
```

Comment:
Need autodoc

Reply:
i have no clue what autodoc means in the slightest so im just going to add a comment explaining what it does

Reply:
i figured it out